TelligentLocalInstance/Initialize.ps1

```diff
@@ -281,4 +305,134 @@ function Write-Telligent
 	Write-Host
 }
 
+function Expand-Zip {
```

Comment:
I guess this applies to other usages too, although we also have a Test-Zip method too.